docs/assets/template-for-community-application.txt

```diff
@@ -0,0 +1,86 @@
+# Template for Community Application
+
+## Message Header and Format:
+
+   Message Header: Community Application_{Name of Community}
+
+- Name of Community or Open Source Project
+- Official Website:
+- Contact: (Name)：
+  - E-mail:
+  - Phone number:
+
+  Please send e-mail to summer@iscas.ac.cn
+
+1. Community name:
+2. Website URL:
+3. Upload Logo
+4. Descriptions 
+These descriptions will be displayed on the organization list page (Short Description) and on your organization's page (Long Description).
+  - Short Description (up to 180 characters)
+
+  - Long Description (may include limited Markdown up to 2000 characters)
+ 
+5. Mailing List ( Generally developers mailing list to help students know the community)

```

Comment:
In order to help students know more about the community, please fill the developers mailing list.